### PR TITLE
Add rcpputils to ros2.repos

### PR DESCRIPTION
**Pull request by emersonknapp**
Related to https://github.com/ros2/ros2/issues/664

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1115078104889039/1115079303997510)

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -195,6 +195,10 @@ repositories:
     type: git
     url: https://github.com/ros2/rcutils.git
     version: master
+  ros2/rcpputils:
```
Comment (thomas-moulard) [inline ros2.repos]: Could you move this after rclpy to keep this alphabetically ordered?

Comment (emersonknapp) [reply ros2.repos]: 👍

**Comment by wjwwood** [comment]
This repository's packages should build before we merge this.

```
Starting >>> rcpputils
--- output: rcpputils
CMake Error: The source directory "/Users/william/ros2_ws/src/ros2/rcpputils" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
---
--- stderr: rcpputils
CMake Error: The source directory "/Users/william/ros2_ws/src/ros2/rcpputils" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
---
Failed   <<< rcpputils	[ Exited with code 1 ]
```

**Comment by emersonknapp** [comment]
@wjwwood Agreed - once we merge this one then it will build https://github.com/ros2/rcpputils/pull/2

**Comment by emersonknapp** [comment]
@thomas-moulard can you trigger CI for this? https://raw.githubusercontent.com/ros2/ros2/f61052f28acfc36bcebe201bcf43ed246e3bcf4e/ros2.repos

**Comment by nuclearsandwich** [comment]
CI build up to rcpputils, test only rcpputils. Let me know if there's anything else we should test on top of this.

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/6436/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/2813/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/5288/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/6262/)

**Comment by emersonknapp** [comment]
@nuclearsandwich no other packages are including it yet, so that build/test is all we need

**Comment by emersonknapp** [comment]
@tfoote Thanks for approving, should we merge?

**Pull request merged**